internal/pkg/template/template_functions.go

```diff
@@ -65,6 +65,9 @@ func EnvVarSecretFunc(s string) string {
 	return StripNonAlphaNumFunc(s) + "Secret"
 }
 
+// Deref dereferences the pointer.
+func Deref(i *string) string { return *i }
```

Comment:
Instead of adding `Deref` which only works for `*string` what do you think of adding a method to the `template.SQSQueue` type:
```go
func (q SQSQueue) IsFIFO() bool
```
which can be used in the template:
```go
{{- if .Susbcribe.Queue.IsFIFO }}
```